release-team/role-handbooks/branch-manager/README.md

```diff
@@ -1,235 +1,307 @@
+
+Important:
+> - [ ] Request to be a [Kubernetes community org member](https://github.com/kubernetes/community/blob/master/community-membership.md#member).
+
+- [ ] Contact [Kubernetes Build Admins][kubernetes-build-admins], to identify yourself as the incoming release branch manager for the current release team and request to be added to the special privilege group for the `kubernetes-release-test` GCP project that's used to build the releases
+   * **NB**: should the release lead also request some type of read access here for themselves?
+   * **NB**: should the branch manager shadows also get at least read access at the beginning?  If a shadow is going to exercise the process on behalf of the lead, they will need full read/write access ahead of time.
+- [ ] Request permission to post on [kubernetes-announce][k-announce-list] via owner contact form [here][k-announce-request]
    * If you haven't received access after 24 hrs, contact the [Release Team][kubernetes-release-team].
-* Development client machine setup:
-   * Linux.  MacOS/Windows are not supported by the scripting today.  However, running the [release tooling] inside a Linux image on MacOS' docker engine works well.
-   * ssh configuration set up for GitHub (either static .ssh/config or ssh agent works)
-   * run `gcloud init` using the identity that you had authorized in the prior bullet point
-   * git clone and checkout the [k/release repo](https://github.com/kubernetes/release) for the `gcbmgr` and `release-notify` scripts
-   * install [Google Cloud SDK](https://cloud.google.com/sdk/install)
-   * ability to run `sendmail` from local Unix command line (BUG: should release-notify script be sending from GCB instead of the local machine, and can it?)
-   * branchff: requires ability for your user to write /usr/local/ directory (BUG),
-     sudo priv's, and membership in https://github.com/orgs/kubernetes/teams/kubernetes-release-managers
-* Join these mailing lists to keep updated:
-   * [kubernetes-sig-release](https://groups.google.com/forum/#!forum/kubernetes-sig-release)
-   * [kubernetes-release-team](https://groups.google.com/forum/#!forum/kubernetes-release-team)
+- Machine setup:
+   - [ ] Linux OS
+      * MacOS/Windows are not supported by [release tools] today. However, running inside a Linux container on MacOS via Docker works well.
+      * See "Cutting v1.15.0-alpha.2" under [References](#References) for an example Dockerfile.
+   - [ ] Setup SSH keys for GitHub (either static .ssh/config or ssh agent works)
+   - [ ] Install the (Google Cloud SDK)[https://cloud.google.com/sdk/docs/quickstart-debian-ubuntu]
+   - [ ] Run `gcloud init` using the identity (i.e. email address) that you had authorized for the GCP project
+   - [ ] `git clone git@github.com:kubernetes/release.git`
+   - [ ] ability to run `sendmail` from local Unix command line
+      * **NB**: should release-notify script be sending from GCB instead of the local machine, and can it?
```

Comment:
This one also.  Let's just drop the TODO/NB and put it into a backlog issue to decide the future.

Reply:
This issue is mentioned in #696.